CONTRIBUTING.md

```diff
@@ -17,3 +17,21 @@ This repo is set to use `pre-commit` to run *isort*, *flake8*, *pydocstring*, *b
 ```bash
 $ pre-commit install
 ```
+
+### Docs
+
+```
+pip install mkdocs mkdocs-material pygments mkapi
```

Comment:
can you add `pygments mkapi` in https://github.com/cogeotiff/rio-tiler-pds/blob/master/setup.py#L14

Reply:
and do `pip install -e .["docs"]`